spec/friendly_shipping/services/tforce_freight/access_token_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe FriendlyShipping::Services::TForceFreight::AccessToken do
+  subject(:access_token) do
+    described_class.new(
+      token_type: "Bearer",
+      expires_in: 3599,
+      ext_expires_in: 3599,
+      access_token: "XYADfw4Hz2KdN_sd8N4TzMo9"
+    )
+  end
+
+  it { is_expected.to respond_to(:token_type) }
+  it { is_expected.to respond_to(:expires_in) }
+  it { is_expected.to respond_to(:ext_expires_in) }
+  it { is_expected.to respond_to(:access_token) }
+
+  describe "#decoded_access_token" do
+    subject(:decoded_access_token) { access_token.decoded_access_token }
```

Comment:
They recommend decoding the token and using the expiration timestamp inside instead of the expiration returned in the response body. I anticipate calling this method from CSpree.

Reply:
Oh, you're just saying the named subject isn't needed in this spec, right? 😄